spec/items_orders.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE items (
+  id SERIAL PRIMARY KEY,
+  name text,
+  price text,
+  quantity int
+);
+
+CREATE TABLE orders (
+  id SERIAL PRIMARY KEY,
+  customer text,
+  order_date text
```

Comment:
Consider using date or datetime instead of text for order_date:

https://learn.microsoft.com/en-us/sql/t-sql/functions/date-and-time-data-types-and-functions-transact-sql?view=sql-server-ver16